client/src/components/settings/__snapshots__/honesty.test.tsx.snap

```diff
@@ -8,7 +8,7 @@ exports[`<Honesty /> <Honesty /> snapshot when isHonest is false: Honesty 1`] =
     settings.headings.honesty
   </SectionHeader>
   <FullWidthRow>
-    <y
+    <b
```

Comment:
Arguably I shouldn't have spent so long tracking this down, but since I did...

What's happening here is that terser is setting a different value for the variable that gets exported as `Panel`. Before it was `y` and now it's `b`.

Quite why that ends up in the snapshot is a mystery for another day, but it's harmless.